.github/workflows/publish-helm.yml

```diff
@@ -0,0 +1,25 @@
+name: "Publish: helm chart"
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'helm/**'
+      - '.github/workflows/publish-helm.yml'
+jobs:
+  publish-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.5.2
```

Comment:
You don't have to pin the version without a specific use case, `setup-helm` will automatically take the latest stable version.

Reply:
I used what has been done in `lint-helm.yml`

Reply:
done